.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve PyTorch
+
+---
+
+## 🐛 Bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
+
+Steps to reproduce the behavior:
+
+1.
+1.
+1.
+
+<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->
+
+## Expected behavior
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Screenshots
```

Comment:
on second-thoughts, I think Screenshots is not relevant. It'll just add a lot of bad false-positives (users taking a picture of the error message, rather than pasting the text)

Reply:
we should remove it